source/src/molecular-unfolding/cdk/stack-main.ts

```diff
@@ -58,6 +59,13 @@ export class MainStack extends SolutionStack {
       ),
     });
 
+    const quickSightUserParam = new CfnParameter(this, 'quickSightUser', {
```

Comment:
add `constraintDescription` property

also add the link to how create QuickSight user in `description`

Reply:
changed